src/main/java/io/nats/client/JetStreamOptions.java

```diff
@@ -63,7 +64,7 @@ public static Builder builder() {
      * @return the configuration
      */
     public static JetStreamOptions defaultOptions() {
-        return new JetStreamOptions(null, null);
+        return DEFAULT_JS_OPTIONS;

```

Comment:
If someone gets the defaults, modifies them, then calls `defaultOptions()` again they would have modified options...  Not sure this is desired behavior.

Reply:
Oh, I see...   there are no setters, only a builder so once created options are immutable.  nevermind.